ts/Dashboard/EditMode/Fullscreen.ts

```diff
@@ -0,0 +1,148 @@
+import Dashboard from '../Dashboard.js';
+import U from '../../Core/Utilities.js';
+
+const { addEvent } = U;
+
+class Fullscreen {
+    /* *
+     *
+     *  Static Functions
+     *
+     * */
+
+    /**
+     * Prepares the dashboard class to support fullscreen.
+     * @param {typeof_Dashboard} DashboardClass
+     *        The dashboard class to decorate with fullscreen support.
+     */
+    public static compose(DashboardClass: Dashboard): void {
```

Comment:
The argument is misleading, because the compose function accepts a `Dashboard` instance, not the class prototype. I suggest to skip the compose and instead move the `DashboardClass.fullscreen = new Fullscreen(DashboardClass);` to the dashboard constructor. So it becomes there `this.fullscreen = new Fullscreen(this);`.